scripts/loadtest-scripts/prepare_loadtests.py

```diff
@@ -2,13 +2,15 @@
 import requests
 import os
 
+from clients.actionclient import ActionClient
 from clients.collectionexerciseclient import CollectionExerciseClient
 from clients.collectioninstrumentclient import CollectionInstrumentClient
 from clients.sampleclient import SampleClient
 
 party_url = os.getenv('PARTY_URL', 'http://localhost:8081')
 party_create_respondent_endpoint = os.getenv('PARTY_CREATE_RESPONDENT_ENDPOINT',
                                              '/party-api/v1/respondents')
+action_url = os.getenv('ACTION_URL', 'http://localhost:????')
```

Comment:
Seems like an odd default, I would rather there be some sort of exception or error message (or no default at all)